hawc/apps/animal/serializers.py

```diff
@@ -11,13 +11,13 @@
 from ..common.helper import SerializerHelper
 from ..common.serializers import get_matching_instance, get_matching_instances
 from ..study.models import Study
-from ..study.serializers import StudySerializer
+from ..study.serializers import VerboseStudySerializer
 from ..vocab.constants import VocabularyTermType
 from . import forms, models
 
 
 class ExperimentSerializer(serializers.ModelSerializer):
-    study = StudySerializer(required=False, read_only=True)
+    study = VerboseStudySerializer(required=False, read_only=True)
```

Comment:
my preference would be to not change to this; the verbose serializer pulls in many more models that aren't required and since they're only used for the exporters which get all the data, it makes the query much more expensive.